bazel/cargo/wasmtime/Cargo.toml

```diff
@@ -11,5 +11,5 @@ path = "fake_lib.rs"
 env_logger = "0.10"
 anyhow = "1.0"
 once_cell = "1.12"
-wasmtime = {version = "23.0.1", default-features = false, features = ['cranelift']}
+wasmtime = {version = "23.0.1", default-features = false, features = ['cranelift', 'runtime', 'gc', 'std']}
```

Comment:
Do we need `gc`?

Reply:
I'm basing its inclusion based on https://github.com/bytecodealliance/wasmtime/blob/dad3d321977a6cff1371ad0b68d90655cf7a9f07/crates/c-api/Cargo.toml#L26